editor/src/messages/portfolio/document/utility_types/network_interface.rs

```diff
@@ -1124,55 +1136,39 @@ impl NodeNetworkInterface {
 		Some(&node.implementation)
 	}
 
-	pub fn input_name<'a>(&'a self, node_id: NodeId, index: usize, network_path: &[NodeId]) -> Option<&'a str> {
```

Comment:
input name no longer relies on the definition name, since it is guaranteed to exist for every input due to `validate_input_metadata` when opening the file